package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "class1",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "./node_modules/webpack/bin/webpack.js",
+    "watch": "./node_modules/webpack-dev-server/bin/webpack-dev-server.js --inline --hot"
```

Comment:
"lint": "./node_modules/.bin/eslint ./**/*.js"


missing the above script, one of the requirements was to have this